ui/src/components/policy/PolicyList.js

```diff
@@ -110,36 +200,47 @@ export default class PolicyList extends React.Component {
         });
     }
 
-    reloadPolicies(successMessage, showSuccess) {
-        let role = this.props.role;
+    onClickDuplicatePolicyVersion(
```

Comment:
This will be called when duplicating the policy ("versions" are not expanded). Duplicated version will have the same rules as the active version.